YAXLib/Customization/IMemberContext.cs

```diff
@@ -11,19 +9,9 @@ namespace YAXLib.Customization;
 public interface IMemberContext
 {
     /// <summary>
-    /// The member's <see cref="MemberInfo" /> for member serialization, else <see langword="null" />.
-    /// </summary>
-    MemberInfo MemberInfo { get; }
```

Comment:
Keep, but mark as "Obsolete, will be removed in a future version"

Reply:
Done. Please take a look.